droplet_actions_test.go

```diff
@@ -291,3 +291,36 @@ func TestDropletActions_Get(t *testing.T) {
 		t.Errorf("DropletActions.Get returned %+v, expected %+v", action, expected)
 	}
 }
+
+func TestDropletAction_Snapshot(t *testing.T) {
```

Comment:
Want to move this up above `TestDropletActions_Get` with the other individual actions?